<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Systemy Wspomagania Decyzji</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="typesLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="typesPushButton">
         <property name="text">
          <string>Cast Types</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="namesLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="namesPushButton">
         <property name="text">
          <string>Change Names</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>186</red>
            <green>186</green>
            <blue>186</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>186</red>
            <green>186</green>
            <blue>186</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>186</red>
            <green>186</green>
            <blue>186</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="uniformRowHeights">
       <bool>false</bool>
      </property>
      <attribute name="headerVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="headerDefaultSectionSize">
       <number>90</number>
      </attribute>
      <attribute name="headerStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionAdd_Row"/>
    <addaction name="actionAdd_Col"/>
    <addaction name="actionDelete_Row"/>
    <addaction name="actionDelete_Col"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionPopulate_from_set"/>
    <addaction name="actionClear"/>
    <addaction name="actionDEBUG"/>
   </widget>
   <widget class="QMenu" name="menuStat">
    <property name="title">
     <string>Stat</string>
    </property>
    <addaction name="actionDyskretyzacjaPRD"/>
    <addaction name="actionDyskretyzacjaNK"/>
    <addaction name="actionStandaryzacja"/>
    <addaction name="actionNormalizacja"/>
    <addaction name="actionOdstajace3x"/>
    <addaction name="actionOdstajaceProcent"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Wykresy</string>
    </property>
    <addaction name="actionWykres2D"/>
    <addaction name="actionWykres3D"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuData"/>
   <addaction name="menuStat"/>
   <addaction name="menuPlot"/>
   <addaction name="menuDebug"/>
  </widget>
  <action name="actionAdd_Row">
   <property name="text">
    <string>Add Row</string>
   </property>
  </action>
  <action name="actionAdd_Col">
   <property name="text">
    <string>Add Col</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>(DEBUG)Clear</string>
   </property>
  </action>
  <action name="actionDelete_Row">
   <property name="text">
    <string>Delete Row</string>
   </property>
  </action>
  <action name="actionDelete_Col">
   <property name="text">
    <string>Delete Col</string>
   </property>
  </action>
  <action name="actionPopulate_from_set">
   <property name="text">
    <string>(DEBUG)Populate</string>
   </property>
  </action>
  <action name="actionDEBUG">
   <property name="text">
    <string>DEBUG</string>
   </property>
  </action>
  <action name="actionEdit_Column_Types">
   <property name="text">
    <string>Edit Column Types</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionDyskretyzacjaPRD">
   <property name="text">
    <string>Dyskretyzacja PRD</string>
   </property>
  </action>
  <action name="actionDyskretyzacjaNK">
   <property name="text">
    <string>Dyskretyzacja NK</string>
   </property>
  </action>
  <action name="actionStandaryzacja">
   <property name="text">
    <string>Standaryzacja</string>
   </property>
  </action>
  <action name="actionNormalizacja">
   <property name="text">
    <string>Normalizacja MinMax</string>
   </property>
  </action>
  <action name="actionOdstajace3x">
   <property name="text">
    <string>Odstajace3x</string>
   </property>
  </action>
  <action name="actionOdstajaceProcent">
   <property name="text">
    <string>OdstajaceProcent</string>
   </property>
  </action>
  <action name="actionWykres2D">
   <property name="text">
    <string>Wykres2D</string>
   </property>
  </action>
  <action name="actionWykres3D">
   <property name="text">
    <string>Wykres3D</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
